plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/detailviews/AbstractStackedDetailViews.java

```diff
@@ -165,6 +171,56 @@ public Composite createControls(Composite parent, Object context, DataBindingCon
 		public boolean isViewFor(Object object) {
 			return false;
 		}
+
+		/**
+		 * Helper method that creates Label + read-only StyledText
+		 * @param labelText
+		 * @param container
+		 * @return
+		 */
+		protected StyledText createLabeledValue(String labelText, Composite container) {
+			Label label = new Label(container, SWT.NONE);
+			label.setText(labelText);
+			GridDataFactory.fillDefaults()
+				.align(SWT.LEFT, SWT.CENTER).applyTo(label);
+			return createValue(container);
+		}
+
+		/**
+		 * Helper method that creates read-only StyledText for a value to display
+		 * @param container
+		 * @return
+		 */
+		protected StyledText createValue(Composite container) {
```

Comment:
imho the method name does not refelct the purpose of this method. It should more be named like #createNonEditableStyledText or something